src/assets/js/react/components/FontManager/FontListItems.js

```diff
@@ -337,8 +350,8 @@ export class FontListItems extends Component {
                 {!disableSelectFontName && (
                   <input
                     type='radio'
-                    className='selectFontName'
-                    name='selectFontName'
+                    className='select-font-name'
+                    name={'select-font-name-' + font.id}
```

Comment:
Add an aria label that describe what does the button do. e.g aria-label="Select {Font name} font".